### ISSUE TITLE
Descent Undo not working correctly

### ISSUE THREAD

**Issue by DelphiDie** [NPBruce/valkyrie#807]
### Description of Problem or Question
If you use the undo option in a Descent Quest the menu bar of MoM is added at the bottom of the screen. Screenshots attached.

### Valkyrie Version
2.0

### Attach Save, Quest or Scenario
Before undo:
![pre undo](https://user-images.githubusercontent.com/26790002/39306327-382f6e0a-4960-11e8-9a9d-a7be573d7f67.jpg)

After undo:
![post undo](https://user-images.githubusercontent.com/26790002/39306339-41d8fcaa-4960-11e8-8c3c-09cb18761484.jpg)

**Comment by DelphiDie** [comment]
I don't know if this would warant a separate issue, but the same thing happens after loading a save in the Descent module.

**Comment by NPBruce** [comment]
Yep, worked that out, I have a fix that will go out after testing.
